src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -132,6 +133,8 @@ internal sealed partial class TexlBinding
         /// </summary>
         internal DName ThisRecordDefaultName => new DName("ThisRecord");
 
+        public Feature Feature { get; }
```

Comment:
Wondering if these flag properties should be named `Features`, assuming we'll have more than one in the future.

Reply:
yes
[https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1714](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1714)